test/uk/gov/hmrc/plasticpackagingtax/returns/builders/TaxReturnBuilder.scala

```diff
@@ -30,7 +31,7 @@ trait TaxReturnBuilder {
     modifiers.foldLeft(modelWithDefaults)((current, modifier) => modifier(current))
 
   private def modelWithDefaults: TaxReturn =
-    TaxReturn(id = "id", obligation = Some(defaultObligation))
+    TaxReturn(id = UUID.randomUUID().toString, obligation = Some(defaultObligation))
```

Comment:
Makes it harder for a test to pass because of something another test did to an un reset mock.